scripts/002-gcc-newlib-PPU.sh

```diff
@@ -44,6 +44,7 @@ fi
 cd ${GCC}/build-ppu
 
 ## Configure the build.
+CFLAGS_FOR_TARGET="-O2" LDFLAGS_FOR_TARGET="" \
```

Comment:
Hm, isn't the default value of `CFLAGS_FOR_TARGET`"-g -O2"?  (Line 2303 in `configure.ac`.)  In which case this does not set any different optimisation, only disable debug information.  I think keeping the debug information would be useful.

Reply:
Yes you're right. in this case, that line is to be removed.